src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocEmptyPeriod.java

```diff
@@ -0,0 +1,24 @@
+/*
+SummaryJavadoc
+violateExecutionOnNonTightHtml = (default)false
+forbiddenSummaryFragments = .*CheckStyle$
+period =
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+// ok
+public class InputSummaryJavadocEmptyPeriod {
+
+    /**
+     * checkStyle. check rule of CheckStyle.
```

Comment:
Add some cases that supposed to match `CheckStyle$`

Reply:
Done

Reply:
I meant such work in  first sentence.

Reply:
Please check